hadoop-worker1/files/etc/supervisord.d/sshd.conf

```diff
@@ -0,0 +1,4 @@
+[program:sshd]
```

Comment:
why not put it to base-ubuntu-2204?

and let's keep other fields like we did in other applications

```
[program:hive-metastore]
directory=/opt/hive
command=hive --service metastore
startsecs=30
stopwaitsecs=10
user=hive
redirect_stderr=true
stdout_logfile=/var/log/hive-metastore.log
autostart=true
```